### ISSUE TITLE
NVDA+F twice fails to work under some circumstances

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#5628]
I'm testing the new feature of NVDA to display the formatting information in a virtual viewer. But in most cases, it does not work. Instead, NVDA plays its error sound. Pressing NVDA+F once reports the formatting correctly as it should. I've tested in MS Word 2013, Mozilla Thunderbird and Mozilla Firefox (both latest stable releases) - it does not work. Strangely enough, it works in Notepad++. I've set NVDA's logging level to "Input/Output" and I've saved the created log file and I'll attach it here.

Steps to reproduce it:
1. Open one of the following applications and load a document in it: MS Word 2013, Mozilla Thunderbird, Mozilla Firefox. It may affect other applications as well, but I haven't tested.
2. Place the focus in the loaded document.
3. Press the command to display the formatting information in a virtual viewer window (NVDA+F twice, by default).

Expected results:
NVDA should display the formatting information in a virtual viewer window.

Actual results:
NVDA plays its error sound.

Test environment:
- Operating system: Windows 10 Pro (build 10586.29), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-13018,cd5ba70, in bulgarian.
- Processor: Intel Core i5-2320 at 3.00GHz.
- RAM Memory: 4.00 GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 2047MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC662 at Intel Cougar Point PCH - High Definition Audio Controller.
[nvda_log_2015-12-22_17-16-03.txt](https://github.com/nvaccess/nvda/files/69861/nvda_log_2015-12-22_17-16-03.txt)
